### DISCUSSION TITLE
Moving data from root to shared buckets without hitting "Transport endpoint is not connected"

### DISCUSSION THREAD

**Discussion by mccabete** [MAAP-Project/Community#820] [Platform] [answered]
I have a list of large files I would like to store in shared-buckets so that other folks on the team can access them. When I try moving them with a simple `mv  myfiles/* my-public-bucket/myfiles/`, I hit a `Transport endpoint is not connected` consistently. I have tried restarting my workspace. 

Is there any solution beyond "Keep trying until all the files get moved"? 

Thanks for the help!

**Comment by mccabete** [comment, marked as answer]
From @wildintellect 

`mamba install -c conda-forge awscli`

aws s3 cp 
https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html

and the s3 bucket for a shared bucket is: 
s3://maap-ops-workspace/shared/_username_/

**Comment by emileten** [comment]
@wildintellect although this seems to be a recurring problem in MAAP (see recent slack conv https://esa-nasa-maap.slack.com/archives/C01308ZHELF/p1697942997820389) and, it seems, elsewhere as well (after a quick internet search), I don't quite have an idea of what the root cause is. Do you ?

**Reply by wildintellect** [reply]
@emileten s3fuse performance issues/bugs

**Reply by emileten** [reply]
for reference, a couple recent issues i found in s3fuse with this error [#2356](https://github.com/s3fs-fuse/s3fs-fuse/issues/2356), [#2176](https://github.com/s3fs-fuse/s3fs-fuse/issues/2176)
